addons/delivery/models/delivery_carrier.py

```diff
@@ -48,6 +48,7 @@ class DeliveryCarrier(models.Model):
     debug_logging = fields.Boolean('Debug logging', help="Log requests in order to ease debugging")
     company_id = fields.Many2one('res.company', string='Company', related='product_id.company_id', store=True, readonly=False)
     product_id = fields.Many2one('product.product', string='Delivery Product', required=True, ondelete='restrict')
+    tracking_url = fields.Char(string='Tracking Link', help="This option adds a link for the customer in the portal to track their package easily. Use <shipmenttrackingNumber> as a placeholder in your URL.")
```

Comment:
Either there is no capital letter, either you write the one for each word
```suggestion
    tracking_url = fields.Char(string='Tracking Link', help="This option adds a link for the customer in the portal to track their package easily. Use <shipmentTrackingNumber> as a placeholder in your URL.")
```

Reply:
done